virttest/vt_cluster/node.py

```
@@ -0,0 +1,455 @@
+        :return: The PID string if found, otherwise None.
+        :rtype: str | None
+        :raises NodeError: If reading the PID file fails.
+        """
+        if self._agent_pid_filename:
+            _session = self._create_session()
+            cmd_open = "cat %s" % self._agent_pid_filename
+            try:
+                pid = _session.cmd_output(cmd_open).strip()
+                if pid:
+                    self._agent_pid = pid
+                return pid
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)